nncf/common/graph/utils.py

```diff
@@ -114,3 +115,17 @@ def get_number_of_quantized_ops(
             else:
                 nodes_to_see.extend(graph.get_next_nodes(node))
     return len(quantized_ops)
+
+
+def get_reduction_axes(axes_to_keep: Union[List[int], ReductionAxes], shape_to_reduce: List[int]) -> ReductionAxes:
```

Comment:
I want to mention that `ReductionAxes` is already a `Tuple[int]`. So I was confused about what put in typehints. It could be any of  `List[int], Tuple[int], ReductionAxes` or `List[int], Tuple[int]` or `List[int], ReductionAxes`?

Reply:
I don't see any reason to use the `ReductionAxes` variable outside the `collectors.py` file since it only confuses.

Reply:
But it is already spread through NNCF

Reply:
This is a problem. Let do not spread problems more.

Reply:
done